content/luau/concepts/conditionals/conditionals.md

```diff
@@ -0,0 +1,89 @@
+---
+Title: 'Conditionals'
+Description: 'Conditionals execute a code block depending on whether an expression is true.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Conditionals'
+  - 'Control Flow'
+  - 'Logic'
+  - 'Logical'
+CatalogContent:
+  - 'learn-lua'
+  - 'introduction-to-game-development'
+---
+
+A **conditional** determines whether or not certain blocks of code will run. A conditional checks the value of an expression, called the **condition**. If the condition is equal to `false`, then the code block within the conditional will be skipped. Otherwise, the code block will be run.
```

Comment:
I believe that in the style guide it recommends using bold for new terms, not italics. Excerpts:

- Bold the first occurrence of the concept or term (the subject) in the body of the entry. _Any additional new concepts/terms introduced in the text are to be in bold as well._ If there is already an existing doc entry for the concept/term, link the term instead of using bold.
- Do not use italics for new terms.